base_tier_validation_formula/__init__.py

```diff
@@ -1 +1,3 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
Please don't add licenses in init files, it only generates bigger (and unneeded) diffs.

Reply:
Removed